index.js

```diff
@@ -202,15 +202,16 @@ const pkg = getPackageJson();
     console.log('newVersion 2:', newVersion);
     newVersion = `${tagPrefix}${newVersion}`;
     console.log(`newVersion after merging tagPrefix+newVersion: ${newVersion}`);
-    console.log(`::set-output name=newTag::${newVersion}`);
+    // Using sh as command instead of directly echo to be able to use file redirection
+    await runInWorkspace('sh', ['-c', `echo "newTag=${newVersion}" >> $GITHUB_OUTPUT`]);
```

Comment:
An alternative would be to write directly to the file referenced by `$GITHUB_OUTPUT`, but this one-liner does the trick and follows the method mentioned in https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.